### ISSUE TITLE
Can't start a game with v1.4.1

### ISSUE THREAD

**Issue by luxzg** [eideehi/valheim-automatics#47]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Mod version

1.4.1

### Is this a bug caused by this mod alone?

The bug was reproduced with this mod alone

### What happened?

I'll be honest, I did not try with just automatics 1.4.1, but only thing I've updated today was Automatics from 1.4.0 to 1.4.1.
When I try to start my game, I can select character, and when I select world to load, and try to load... simply nothing happens. I can click the button all I want.
I've noticed error in console mentions Automatics, so I've copied it below.

Renaming `Automatics.dll` allowed game to start without issues.

I have 60+ mods, but until today they've all played nicely with each others, and none of them had issues with Automatcis 1.4.0

I only have auto-doors "module" enabled, if that helps.

### Steps to reproduce the bug

Upgrade v.1.4.0 to v1.4.1. Try to start a game - no go.
Disable dll - game starts fine

### Logs and screenshots

Log:

```
[Info   : Unity Log] 03/11/2023 20:03:26: show character selection

[Info   : Unity Log] 03/11/2023 20:03:27: OnCharacterStart

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Automatics.Patches+<>c.<FejdStartup_Awake_Postfix>b__0_0 (System.Object sender, FejdStartup+StartGameEventArgs args) (at <22c7936e25484513ae594b929358d2da>:0)
FejdStartup.OnWorldStart () (at <0a1a436be26b421bb46752e573a7aa2f>:0)
UnityEngine.Events.InvokableCall.Invoke () (at <c9fb31c2eee14ce4aeb36e4c61e154d8>:0)
UnityEngine.Events.UnityEvent.Invoke () (at <c9fb31c2eee14ce4aeb36e4c61e154d8>:0)
UnityEngine.UI.Button.Press () (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.UI.Button.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.EventSystem:Update()
```

**Comment by eideehi** [comment]
This bug has been fixed with commit 04d6bd46af47aadc0c0eed1b1bf0349061b9e8ee. Please wait for the bug fix release.

**Comment by luxzg** [comment]
Works with 1.4.2 now, thank you!
